.circleci/config.yml

```diff
@@ -52,47 +59,47 @@ jobs:
 commands:
   setup_system:
     steps:
-       - run:
-           name:  "Setup system environment"
-           command: |
-             wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
-             curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
-             echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
-             sudo apt-get update
-             sudo apt-get install -y libzmq3-dev
+      - run:
+          name:  "Setup system environment"
+          command: |
+            wget -q -O - https://dl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+            curl -sL https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+            echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+            sudo apt-get update
+            sudo apt-get install -y libzmq3-dev
   setup_nodejs:
     steps:
-       - run:
-           name:  "Install nodejs"
-           command: |
-             set +e
-             curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.33.5/install.sh | bash
-             export NVM_DIR="/opt/circleci/.nvm"
-             [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
-             nvm install v10.14.2
-             nvm alias default v10.14.2
+      - run:
+          name:  "Install nodejs"
```

Comment:
I think instead of this we should just install `node` via `apt` as we did in travis. This avoids all this shell env stuff.